### PR TITLE
Translate ruby 2.3.0-preview2 news (ko)

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

**Comment by marocchino** [comment]
@ruby/www-ruby-lang-org-i18n-ko review please.

File: ko/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+title: "루비 2.3.0-preview2 릴리스"
+author: "naruse"
+translator: "marocchino"
+date: 2015-12-11 14:00:00 +0000
+lang: ko
+---
+
+루비 2.3.0-preview2가 릴리스되었습니다.
+
+루비 2.3.0-preview2는 루비 2.3.0의 두번째 브리뷰입니다.
```
Comment (yous) [inline ko/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md]: 두번째 브리뷰 → 두 번째 프리뷰

File: ko/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+[RubyVM::InstructionSequence#to_binary와
+.load_from_binary](https://bugs.ruby-lang.org/issues/11788)가 실험적인 기능으로
+도입되었습니다. 이 기능으로 ISeq(바이트코드) 전처리 시스템을 만들 수 있습니다.
+
+루비 2.3에는 많은 성능 향상이 포함됩니다.
+예를 들어, [Proc#call 최적화](https://bugs.ruby-lang.org/issues/11569),
+[메소드 엔트리 데이터 구조 재고](https://bugs.ruby-lang.org/issues/11278),
+[테이블 데이터 구조 도입](https://bugs.ruby-lang.org/issues/11420),
+객체 할당과 메소드 호출 코드의 머신 코드 레벨 튜닝과 그밖에 수많은 최적화가
```
Comment (yous) [inline ko/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md]: 그밖에 → 그 밖의

**Comment by yous** [comment]
I'm done.

**Comment by marocchino** [comment]
@yous Thanks!

**Comment by stomar** [comment]
Note the command line option had been changed (fixed with 15ee1bc837b83e3d76b89f056ff472782ecc2cc0).

**Pull request merged**